packages/component-library/draft/Kaizen/SplitButton/styles.scss

```diff
@@ -0,0 +1,171 @@
+@import "~@cultureamp/kaizen-component-library/styles/color";
+@import "~@cultureamp/kaizen-component-library/styles/type";
+@import "~@cultureamp/kaizen-component-library/styles/border";
+@import "~@cultureamp/kaizen-component-library/styles/layers";
+
+$base-border: 1px solid $ca-border-color;
+$focus-border-width: 2px;
+$button-width: 124px;
+
+.root {
+  display: inline-flex;
+}
+
+.buttonReset {
+  appearance: none;
+  display: inline;
+  background: transparent;
+  color: inherit;
+  font: inherit;
+  cursor: pointer;
+  margin: 0;
+  padding: 0;
+  border: 0;
+}
+
+.buttonStates {
+  &:hover {
+    text-decoration: none;
+    color: inherit;
+  }
+
+  &:not(.disabled):not(:disabled) {
+    background: $white;
+    border-color: add-tint($ca-palette-ink, 80%);
+    color: $ca-palette-ink;
+
+    &:hover,
+    &:active {
+      background: $ca-palette-stone;
+      border-color: add-tint($ca-palette-ink, 60%);
+    }
+
+    &:active {
+      background: $ca-palette-ash;
+      border-color: add-tint($ca-palette-ink, 40%);
+      transform: translateY(1px);
+    }
+
+    // Use JS polyfill to simulate :focus-visible, not yet supported by browsers
+    // https://github.com/WICG/focus-visible#backwards-compatibility
+    :global(.js-focus-visible) & {
+      // hide native focus ring when :focus but not :focus-visible
+      &:focus {
+        outline: none;
+      }
+
+      // show custom focus ring when :focus-visible
+      &:global(.focus-visible) {
+        border: $focus-border-width solid $ca-palette-ocean;
```

Comment:
Most of the code here was pasted from murmur. There are a couple of minor things that I noticed, that I didn't spend time on, but let me know if I should clean it up.

One of the things was this line. When the user tabs onto the SplitButton, the added 2px border will change the size of the component. Ideally I'd say the size should remain the same.